cleanup.php

```
@@ -1,51 +1,58 @@
 #!/usr/bin/env php
-
 <?php
+if (php_sapi_name() !== 'cli') {
+    http_response_code(403);
+    exit();
+}
 
-    require_once 'config.php';
-    require_once 'db_helper.php';
-    require_once 'osm_photo_note.php';
+require_once 'db_helper.php';
+require_once 'osm_photo_note.php';
 
-    $db_helper = new DBHelper();
+$db_helper = new DBHelper();
 
-    $old_inactive_photos = $db_helper->get_and_delete_old_inactive_photos();
-    foreach($old_inactive_photos as $photo) {
-        $file_name = $photo['file_id'] . $photo['file_ext'];
-        unlink($PHOTOS_TMP_DIR . '/' . $file_name);
+function deleteFile($file_path)
+{
+    echo "Deleting $file_path\n";
```

stray echo?

---

Also, I think it is completely reasonable that a warning is emitted when the file does not exist, why this check?

---

Console output for logging or debugging, optional.

---

```php
function deleteFile($file_path)
{
    if (file_exists($file_path)) {
        echo "Deleting $file_path\n";
        unlink($file_path);
    } else {
        echo "$file_path not found\n";
    }
}
```

---

I mean, you replaced a warning when a file could not be deleted (emitted by `unlink`) with always logging when a file is deleted but not warning if the deletion failed.

---

Why not 
```
unlink($file_path);
```
then?

---

Normally the error reporting for PHP is disabled in production, so `unlink` would fail silently.

---

It is executed on command line, I don't think error reporting is disabled for php-cli. Besides, I don't get the stack trace, only the first line, which is fine. So please keep this using the simple `unlink`.